sandbox.html

```diff
@@ -249,6 +257,7 @@
           <optgroup label="Without Iframe">
             <option value="downloadData">Download data</option>
             <option value="fetchData">Fetch data</option>
+            <option value="getData">Get data</option>            
```

Comment:
Should it be in 'General' then?

Reply:
I wanted to be helpful with the grouping, because we're adding more and more actions. But maybe it's just confusing, feel free to refactor or remove the grouping again. But "Without Iframe" is definitely not correct.

Reply:
I think it is nice.
I added PostMessage Interactions but I am also not sure if this should refer to the implementation detail (postMessage)